mbf_msgs/srv/CheckPath.srv

```diff
@@ -21,6 +21,6 @@ uint8                      UNKNOWN   =  3    # path is partially in unknown spac
 uint8                      OUTSIDE   =  4    # path is partially outside the map
 
 uint32                     last_checked      # index of the first pose along the path with return_on state or worse
-uint8                      state             # path worst state (until last_checked): FREE, INFLATED, LETHAL, UNKNOWN...
+uint8                      state             # path worst state (until last_checked): FREE, INSCRIBED, LETHAL, UNKNOWN...
```

Comment:
OOS: make comment match the constant name
This should not change checksum: http://wiki.ros.org/ROS/Technical%20Overview#Message_serialization_and_msg_MD5_sums

(2 more)